web/satellite/src/views/forgotPassword/ForgotPassword.vue

```diff
@@ -13,6 +13,8 @@ import LogoIcon from '@/../static/images/Logo.svg';
 
 import { AuthHttpApi } from '@/api/auth';
 import { RouteConfig } from '@/router';
+import { NOTIFICATION_ACTIONS } from '@/utils/constants/actionNames';
```

Comment:
This import has been added but there is any new line or a changed one that uses it.